defaults.go

```diff
@@ -85,6 +86,13 @@ var DefaultEnableRelay = func(cfg *Config) error {
 	return cfg.Apply(EnableRelay())
 }
 
+var DefaultResourceManager = func(cfg *Config) error {
+	limiter := rcmgr.NewStaticLimiter(0.125, 128<<20, 1<<30)
```

Comment:
maybe add a comment what this is in human readable units? (128 MB and 1 GB, if I'm not mistaken?)

Reply:
yeah, good point, will do.

Reply:
done.